lib/kafka_ex/protocol/produce.ex

```diff
@@ -34,7 +34,7 @@ defmodule KafkaEx.Protocol.Produce do
   def create_request(
     correlation_id,
     client_id,
-    request = %Request{compression: compression, messages: messages}
+    %Request{compression: compression, messages: messages} = request
```

Comment:
I personally prefer the other way - `var = %Struct{...}`... credo just looks for consistency iirc?

Reply:
It does look for consistency, but I think the elixir standard library always does `%Struct{} = var`.

Reply:
🤷‍♂️ I don't care either way, but I did switch a bunch to the `%Struct{} = var` in #232 😱